vector/src/appfdroid/java/im/vector/push/fcm/NotificationTroubleshootTestManagerFactory.kt

```diff
@@ -33,11 +33,11 @@ class NotificationTroubleshootTestManagerFactory {
             if (session != null) {
                 mgr.addTest(TestBingRulesSettings(fragment, session))
             }
-            mgr.addTest(TestNotificationServiceRunning(fragment))
-            mgr.addTest(TestServiceRestart(fragment))
+            // mgr.addTest(TestNotificationServiceRunning(fragment))
+            // mgr.addTest(TestServiceRestart(fragment))
             mgr.addTest(TestAutoStartBoot(fragment))
```

Comment:
Do we still need start on boot?

Reply:
Yes, I think for FDroid we need to simulate a Push on Reboot